src/gp/src/GPMSA.C

```diff
@@ -968,6 +972,9 @@ GPMSAFactory<V, M>::setUpEmulator()
     (*BT_Wy_B_inv)(i,i) +=
       this->m_opts->m_observationalCovarianceRidge;
 
+  // Emulator expects the error matrix, so invert Wy to get it.
+  m_observationErrorMatrix.reset(new M(Wy.inverse()));
+
```

Comment:
Thanks!  The inefficiency doesn't bother me too too much (since it's once per run, not once per sample), but you had me questioning my sanity.  "Is there something wrong with my basic understanding of linear algebra identities?  Is the inverse-of-the-inverse somehow better for some numerical reason?"